caddyhttp/requestid/requestid.go

```diff
@@ -0,0 +1,43 @@
+package requestid
+
+import (
+	"fmt"
+
+	"github.com/mholt/caddy"
+	"github.com/nu7hatch/gouuid"
```

Comment:
This dependency will have to be vendored. Use https://github.com/FiloSottile/gvt to do it.